framework/src/imgui_manager.cpp

```diff
@@ -76,7 +76,27 @@ namespace gvk
 			}
 			mRenderpass = context().create_renderpass(
 				attachments,
-				[](avk::renderpass_sync& rpSync){
+				[](avk::renderpass_sync& rpSync) {
+					if (rpSync.is_external_pre_sync()) {
+						rpSync.mSourceStage = avk::pipeline_stage::color_attachment_output;
+						rpSync.mSourceMemoryDependency = avk::memory_access::color_attachment_write_access;
+						rpSync.mDestinationStage = avk::pipeline_stage::color_attachment_output;
+						rpSync.mDestinationMemoryDependency = avk::memory_access::color_attachment_read_access;
+					}
+					if (rpSync.is_external_post_sync()) {
+						rpSync.mSourceStage = avk::pipeline_stage::color_attachment_output;
+						rpSync.mSourceMemoryDependency = avk::memory_access::color_attachment_write_access;
+						rpSync.mDestinationStage = avk::pipeline_stage::bottom_of_pipe;
+						rpSync.mDestinationMemoryDependency = {};
+					}
+				}
+			);
+
+			// setup render pass for the case where the invokee does not write anything on the backbuffer (and clean it)
+			attachments[0] = avk::attachment::declare(format_from_window_color_buffer(wnd), avk::on_load::clear, avk::color(0), avk::on_store::store);
+			mClearRenderPass = context().create_renderpass(
```

Comment:
I appreciate such additions. I am afraid that `imgui_manager` still requires a lot of work in general. So everything that improves it is great!

Reply:
This was also necessary to allow invokees to be disabled, otherwise vulkan wouldn't like the present format remaining from the previous GUI render.